src/clp_logging/handlers.py

```
@@ -996,3 +1012,282 @@ def _serialize_kv_pair_log_event(
+        self.last_timestamp_ms: int = floor(time.time() * 1000)  # convert to ms and truncate
+        self._ostream.write(
+            FourByteEncoder.encode_preamble(
+                self.last_timestamp_ms, self.timestamp_format, self.timezone
+            )
+        )
+
+    def _remote_log_naming(self) -> str:
+        """
+        Set the name of the target S3 object key to upload to
+        """
+        self._remote_folder_path: str = f"{self.s3_directory}{self._start_timestamp.year}/{self._start_timestamp.month}/{self._start_timestamp.day}"
```

Ruff (0.8.2)

Line too long (148 > 100)

(E501)